src/com/vivian/java/leetcode/Q125_ValidPalindrome.java

```diff
@@ -8,5 +8,52 @@
  * is not a palindrome.
  */
 public class Q125_ValidPalindrome {
+    public boolean validPalindrome(String str) {
+        if (str.isEmpty() || str == null) {
+            System.out.println("not a palindrome");
```

Comment:
System.out.println should be used for debug only. Remove it before you commit to GitHub.